### PR TITLE
#4902 improve participants local time display

### PR DESCRIPTION
**Pull request by dklymenk**
### Details

Updates the `shouldShowReportRecipientLocalTime` value expression to compare UTC offset instead of raw timezone names.

### Fixed Issues

$ https://github.com/Expensify/App/issues/4902

### Tests / QA Steps

Assume that timezone A and timezone B have the same UTC offset, but different names. Example: Europe/Moscow and Europe/Istanbul (both are UTC +3).

1. Log in with account A
2. Go to profile settings
3. Manually set timezone to timezone A
4. Log in with account B
5. Go to profile settings
6. Manually set timezone to timezone B
7. Open chat with account A
8. The recipients local time should not be shown since both timezones have the same UTC offset.

### Tested On

- [x] Web
- [x] Mobile Web
- [x] Desktop
- [x] iOS
- [x] Android

### Screenshots

On screenshots below I have timezone Europe/Istanbul and my participant has Europe/Moscow. Both are UTC +3 and participants local time is not shown.

#### Web

DeepinScreenshot_select-area_20210909202636

#### Mobile Web

Simulator Screen Shot - iPhone 12 - 2021-09-09 at 20 53 47

#### Desktop

Screen Shot 2021-09-09 at 20 56 32

#### iOS

Simulator Screen Shot - iPhone 12 - 2021-09-09 at 20 51 12

#### Android

Screenshot_1631210201

### CODE REVIEW

File: src/pages/home/report/ReportActionCompose.js [hunk 1]
```diff
@@ -268,6 +269,16 @@ class ReportActionCompose extends React.Component {
         return this.props.translate('reportActionCompose.writeSomething');
     }
 
+    /**
+     * Get timezone's UTC offset in minutes
+     *
+     * @param {String} [timezone]
+     * @return {Number}
+     */
+    getTimezoneOffset(timezone) {
```
Comment (stitesExpensify) [inline src/pages/home/report/ReportActionCompose.js:278]: Do we need this or can we just call the moment method directly below?

Comment (dklymenk) [reply src/pages/home/report/ReportActionCompose.js:278]: We can do it directly below. We can also move this function to date utils if you don't want to pollute the methods of the class.

Comment (stitesExpensify) [reply src/pages/home/report/ReportActionCompose.js:278]: I just don't think we need the method in general tbh. It's just calling another method that already exists

Comment (dklymenk) [reply src/pages/home/report/ReportActionCompose.js:278]: Ok, I will delete that method and do the moment stuff directly.

Comment (dklymenk) [reply src/pages/home/report/ReportActionCompose.js]: Done

**Review by stitesExpensify — APPROVED** [review]

**Comment by botify** [comment]
This has been deployed to production and is now subject to a 7-day regression period.
If no regressions arise, payment will be issued on 2021-09-22.  :confetti_ball:

**Pull request merged**